device-orientation.html

```diff
@@ -66,6 +75,10 @@
           beta: event.beta,
           gamma: event.gamma
         });
+
+        this._setSensorsAvailable(
+          !(event.alpha === null && event.beta === null && event.gamma === null)
```

Comment:
I mean, is check for `null` is enough? Should we check for `undefined` and `""` (the empty string)?